server/src/main/java/org/jboss/as/server/mgmt/domain/HostControllerConnectionService.java

```diff
@@ -81,7 +80,7 @@
 
     private final URI connectionURI;
     private final String serverName;
-    private final String userName;
+    private final String userName = null; // TODO This likely needs to be further visited.
```

Comment:
JIRA?

Reply:
I believe this one is under this issue to follow up https://issues.redhat.com/browse/WFCORE-5544